README.md

```diff
@@ -111,7 +111,7 @@ To organize and steer the development of [helm-operator-plugins](https://github.
 
 ### Meetings
 
-* Issue Triage and Community Meeting: [Tuesdays at 11:00 ET (Eastern Time)](https://github.com/operator-framework/helm-operator-plugins/issues?q=is%3Aopen+is%3Aissue+no%3Amilestone+sort%3Acreated-asc) (biweekly). [Convert to your timezone](http://www.thetimezoneconverter.com/?t=11:00&tz=ET%20%28Estern%20Time%29).
+* Issue Triage and Community Meeting: [Tuesdays at 11:00 ET (Eastern Time)](https://github.com/operator-framework/helm-operator-plugins/issues?q=is%3Aopen+is%3Aissue+no%3Amilestone+sort%3Acreated-asc) (biweekly). [Convert to your timezone](http://www.thetimezoneconverter.com/?t=11:00&tz=ET%20%28Eastern%20Time%29).
```

Comment:
How did I miss this? I guess I assumed the URL was copied in so I didn't actually read it.